### PR TITLE
Awesome books project - Milestone 2

### PR DESCRIPTION
**Pull request by charlyeneh**
Hello Reviewer,

In this milestone we did:

- The use of classes was implemented in refactoring the code.
- We also added some styling to the CSS file.
- No linter errors
- Make the design almost the same as the given
- Fixed reviewer issues

Thanks 😁

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,81 +1,78 @@
...
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  static removeBook(ref) {
+    const result = books.filter((value) => value.id !== ref);
+    books = result;
+    addToLocalStorage(books);
+  }
```
Comment (er-contreras) [inline script.js]: - [x] The remove button is not working at all. Please kindly fix it

File: style.css [hunk 2]
```diff
@@ -31,10 +110,63 @@ p {
...
+
+.book_lists {
+  display: flex;
+  justify-items: center;
+  padding: 1rem;
+  gap: 0.2rem;
+  width: 35rem;
+  margin: 0 3rem 0 35rem;
+  border: 3px solid black;
+}
```
Comment (er-contreras) [inline style.css]: - [x] All the classes 'book_lists' seem to stay in one place and that does not let them be responsive. 
Screen Shot 2022-02-09 at 16 35 17

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 3]
```diff
@@ -31,51 +36,44 @@ function addEvents() {
...
   });
 }
+
 // populate dom with the list
 const addBtn = document.querySelector('#add-btn');
 function showBooks(list) {
-  const booksListDiv = document.querySelector('#books-container');
-  list.forEach((lists, i) => {
+  const booksListDiv = document.querySelector('#List');
+  // let colorChange = 0;
```
Comment (Meltrust) [inline script.js]: - [ ] Kindly please, remove this introduced old piece of commented-out code.  That way, the project will be cleaner 💪

File: script.js [hunk 4]
```diff
@@ -31,51 +36,44 @@ function addEvents() {
...
-    div.appendChild(pAuthor);
     const btn = document.createElement('button');
     btn.className = 'remove';
     btn.innerHTML = 'Remove';
     btn.id = list[i].id;
     div.appendChild(btn);
-    const hr = document.createElement('hr');
-    div.appendChild(hr);
+    // const hr = document.createElement('hr');
+    // div.appendChild(hr);
```
Comment (Meltrust) [inline script.js]: - [ ] Kindly please, remove this introduced old piece of commented-out code.  That way, the project will be cleaner 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @charlyeneh, 

## Great job fixing the delete button!


Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS APPROVED ✔️ 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**